SPECS/ig/ig.spec

```
@@ -0,0 +1,51 @@
+Summary:        The eBPF tool and systems inspection framework for Kubernetes, containers and Linux hosts.
+Name:           ig
+Version:        0.25.0
+Release:        1%{?dist}
+License:        Apache 2.0 and GPL 2.0 for eBPF code
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+Group:          Tools/Container
+URL:            https://github.com/inspektor-gadget/inspektor-gadget
+Source0:        https://github.com/inspektor-gadget/inspektor-gadget/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-govendor-v1.tar.gz
+BuildRequires:  golang
+
+
+%description
+Inspektor Gadget is a collection of tools (or gadgets) to debug and inspect Kubernetes resources and applications. It manages the packaging, deployment and execution of eBPF programs in a Kubernetes cluster, including many based on BCC tools, as well as some developed specifically for use in Inspektor Gadget. It automatically maps low-level kernel primitives to high-level Kubernetes resources, making it easier and quicker to find the relevant information.
+
+This package contains ig, the local CLI flavor of Inspektor Gadget.
+
+%prep
+%autosetup -n inspektor-gadget-%{version}
+%setup -q -n inspektor-gadget-%{version} -T -D -a 1
+
+%build
+CGO_ENABLED=0 go build \
+		-ldflags "-X github.com/inspektor-gadget/inspektor-gadget/cmd/common.version=v%{version} \
+			-X github.com/inspektor-gadget/inspektor-gadget/cmd/common/image.builderImage=ghcr.io/inspektor-gadget/ebpf-builder:v%{version} \
+			-extldflags '-static'" \
+		-tags "netgo" \
+		-o ./bin/build/ig ./cmd/ig
+
+%install
+mkdir -p "%{buildroot}/%{_bindir}"
+install -D -m0755 bin/build/ig %{buildroot}/%{_bindir}
+
+%check
+set -e
```

To be sure we fail when one of the commands fail:
```suggestion
set -e
set -o pipefail
```